src/charm.py

```diff
@@ -215,11 +215,12 @@ def reconfigure_nats(self):
                 self.ca_client.ca_certificate.public_bytes(encoding=serialization.Encoding.PEM).decode('utf-8'))
 
         if self.nrpe_client.is_available:
+            check_name = "check_{}".format(self.model.unit.name.replace("/", "_"))
```

Comment:
We should still indicate what it checks, aka tcp here
something like

```python
check_name = "check_tcp_{}".format(self.model.unit.name.replace("/", "_"))
```

Reply:
Do we? We adding implementation details to the check name. How the check works should be not in the name as we can easily change the implementation at any time without changing the check name